packages/material-ui/src/CardHeader/CardHeader.js

```diff
@@ -57,6 +57,7 @@ function CardHeader(props) {
     title = (
       <Typography
         variant={avatar ? 'body2' : 'headline'}
+        suppressDeprecationWarnings
         className={classes.title}
```

Comment:
Deprecation warnings are a hint for the user so that they know where a breaking change will happen in the next major release.

A breaking change will happen in that component. The user will not be warned about this. This is bad.

Reply:
Isn't a deprecation warnings noise for the users when it's not actionable?

Reply:
It is actionable since it will be a breaking change in the next major relase. See https://semver.org/#how-should-i-handle-deprecating-functionality.

Reply:
It's not part of the public API.

Reply:
@oliviertassinari  They are by not switching to a typography v2 theme if they're using this component. It's all about where the behavior of software will change. Users can't deal with per instance because we decided against that.